_pages/ato/ssp.md

```diff
@@ -0,0 +1,95 @@
+---
+title: System Security Plan
+---
+
+As described in [the NIST guide](http://csrc.nist.gov/publications/nistpubs/800-18-Rev1/sp800-18-Rev1-final.pdf#page=7):
+
+> The purpose of the system security plan is to provide an overview of the security requirements of the system and describe the controls in place or planned for meeting those requirements.
+
+GSA provides a [Word template](http://www.gsa.gov/graphics/staffoffices/System_Security_Plan_Template_072312_508.docx), but for 18F's purposes, we create a machine-readable version.
+
+We are working on a tool to help generate SSPs – check out [Compliance Masonry](https://github.com/opencontrol/compliance-masonry).
+
+### Template
+
+Add the following to your repository as `system-security-plan.yml`, with the information in the `<angle brackets>` filled in:
+
+```yaml
+---
+name: <project name>
+uniqueID: <MB number – see https://docs.google.com/spreadsheets/d/1hjCYIskgD_x_MI1ehXoiz2Qvsyxj1yK3fxabkezMPiE/edit#gid=0>
+version: <application version>
```

Comment:
@NoahKunin Any context for this value? Was a it determined by someone on the project team?